Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -108,11 +129,13 @@ def is_nested(self):
 
 
 class MemberVariable(object):
```

Comment:
Yeah, I was thinking of adding a new class, but we'd need similar set of properties to avoid random runtime errors here. Also, it does look like that would cause confusion around what object is stored inside `members = []`.

We could have a new container and a new class though, but might be overkill. 

Thoughts?